### PR TITLE
Issue521 adjusted handling of commandline options to preserve case of custom input

### PR DESCRIPTION
**Pull request by hansenmc**
I adjusted the regex to add an additional match group and ensure that .upcase is only applied to the base option name and not the custom input. This will ensure that custom inputs are applied properly when specified on the commandline.

### CODE REVIEW

**Comment by grtjn** [comment]
I think this could use a git rebase, bit hard to distinguish what is the real change here..

File: deploy/lib/server_config.rb [hunk 1]
```diff
@@ -1182,11 +1182,11 @@ def corb
     end
 
     # collect options with either "--" or "-D" prefix, and normalize options to be UPPER-CASE
-    optionArgPattern = /^(--|-D)(.*?)="?(.*)"?/
+    optionArgPattern = /^(--|-D)(([^.]*?)(\..*?)?)="?(.*)"?/
```
Comment (grtjn) [inline deploy/lib/server_config.rb]: You have nested groups in here, which you probably don't need..

Comment (hansenmc) [inline deploy/lib/server_config.rb]: Agreed, I'll remove the wrapping group

**Comment by grtjn** [comment]
Hmm, still too many commits in this PR. I think that is because this PR is against master branch instead of dev.

@hansenmc Could you open a new PR with your changed, but against dev branch?

**Comment by hansenmc** [comment]
Whoops. I'll take a look tonight and get it cleaned up. 

Also, I did commit changes for better usage statements in CoRB2 to our development branch. There are a handful of little enhancements and bug fixes that are ready to ship, so I'll probably put out a point release and have this reference the new version, unless @bbandlamudi wants to hold off.

**Comment by bbandlamudi** [comment]
Hi Mads, if we have enough changes for 2.3.2, we can release it. We need to think of new additions and I would like to hash them out with you on email.

**Pull request closed without merging**